<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightControl</class>
 <widget class="QWidget" name="LightControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LightControl</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>光源端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ledtLightCom"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>操作指令</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbboxCommand">
       <item>
        <property name="text">
         <string>打开通道</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>关闭通道</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>调整亮度</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>读取亮度</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>通道</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbboxLightChannel">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>数值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ledtLightValue"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="ledtInfo">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="pbtnOpenCom">
       <property name="text">
        <string>打开端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtnCloseCom">
       <property name="text">
        <string>关闭端口</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pbtnSet">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtnSave">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
